src/app/component/eco-news/eco-news.module.ts

```diff
@@ -25,6 +26,8 @@ import { EffectsModule } from '@ngrx/effects';
 import { EcoNewsEffects } from './store/eco-news.effects';
 import { EcoNewsSelectors } from './store/eco-news.selectors';
 import { CommentsModule } from '../comments/comments.module';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
```

Comment:
Please, sort imports in such order: angular files, core files/libraries (RxJS etc.), local files

Reply:
fixed